### PR TITLE
(test) Fix warnings raised when running tests

### PR DESCRIPTION
**Pull request by denniskigen**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide).
- [x] My work includes tests or is validated by existing tests.

## Summary

This PR silences a bunch of warnings that get raised when running tests in this repository.

## Screenshots

Screenshot 2022-04-05 at 16 10 10

Screenshot 2022-04-05 at 12 31 43

### CODE REVIEW

**Comment by denniskigen** [comment]
Very curious to know if anyone has a clue as to why these warnings appear:

Screenshot 2022-04-05 at 22 16 02

**Comment by brandones** [comment]
> Very curious to know if anyone has a clue as to why these single-spa-related warnings appear

It would appear that single-spa is being loaded. I don't know where or why.

File: packages/esm-patient-notes-app/src/notes/notes-detailed-summary.test.tsx [hunk 1]
```diff
@@ -115,7 +115,7 @@ describe('NotesDetailedSummary: ', () => {
       expect(within(table).getByRole('row', { name: new RegExp(row, 'i') })).toBeInTheDocument(),
     );
 
-    expect(screen.getAllByRole('row').length).toEqual(6);
+    expect(screen.getAllByRole('row').length).toEqual(11);
```
Comment (brandones) [inline packages/esm-patient-notes-app/src/notes/notes-detailed-summary.test.tsx:118]: Why the functional change?

**Review by brandones — APPROVED** [review]
Looks good, just one question

**Pull request merged**